### DISCUSSION TITLE
Disable SIP Big Sur

### DISCUSSION THREAD

**Discussion by maishams** [asmvik/yabai#1015] [Q&A]
Hi all!

When I go into recovery mode I get a prompt to choose a user to recover the password for. In the menu bar, there is no other spaces apart from user recovery. If I click on the apple logo I can choose a startup disk to start from but that's it -- I can't get to the terminal. Any ideas on anything I could be missing (I'm using a work laptop but I have complete admin control)?

**Comment by robrecord** [comment]
You need to first choose a user and provide its password. After this you will have the options described.
